packages/manager/modules/pci/src/projects/new/payment/payment.controller.js

```diff
@@ -2,10 +2,20 @@ import find from 'lodash/find';
 
 export default class PciProjectNewPaymentCtrl {
   /* @ngInject */
-  constructor($q, $translate, CucCloudMessage, ovhPaymentMethod, PciProjectNewService) {
+  constructor(
+    $q,
+    $state,
+    $translate,
+    atInternet,
+    CucCloudMessage,
+    ovhPaymentMethod,
+    PciProjectNewService,
+  ) {
     // dependencies injections
     this.$q = $q;
+    this.$state = $state;
```

Comment:
I though that `$state` was only injected in routing file?

Reply:
No, `$state` is already injected here :)